src/framework/COScript.m

```diff
@@ -407,11 +407,25 @@ - (void)deleteObjectWithName:(NSString*)name {
 
 # pragma mark - require
 
-- (NSURL*)resolveModule:(NSString *)module currentURL:(NSURL*)currentURL {
+// This aims to implement the require resolution algorithm from NodeJS.
+// Given a `module` string required by a given script at the `currentURL`
+// using `require('./path/to/module')`, this method returns a URL
+// corresponding to the `module`.
+// `module` could also be the name of a core module that we shipped with the app
+// (for example `util`), in which case, it will return the URL to that one
+// and set `isRequiringCore` to YES.
+- (NSURL*)resolveModule:(NSString*)module currentURL:(NSURL*)currentURL isRequiringCoreModule: (BOOL*)isRequiringCore {
```

Comment:
I have to admit I meant adding spaces! 😆

Reply:
hum that matches the style of the rest of the style. Maybe we need a linter for the entire codebase but that's probably not the place nor the time haha

Reply:
ah, good point. I suppose CocoaScript is free to favour whatever convention it likes.